data_io/connection.py

```diff
@@ -449,7 +452,8 @@ def read(
             # for multiple people?
             # XXX This should be set through a shared configuration
             # file so other groups can use it
-            tempdir='s3a://tfsds-lsg-test/ingestion/redshift_temp',
+            # tempdir='s3a://tfsds-lsg-test/ingestion/redshift_temp',
+            tempdir='s3a://tfsdl-lsg-lslpgds-test/redshift',
```

Comment:
looks good, new s3 url matches the email

Reply:
This piece looks good, thank you